.devcontainer/Dockerfile

```
@@ -0,0 +1,47 @@
+COPY --from=graalvm_installer /opt/graalvm-jdk /opt/graalvm-jdk
+ENV JAVA_HOME="/opt/graalvm-jdk"
+ENV GRAALVM_HOME="/opt/graalvm-jdk"
+ENV PATH="$GRAALVM_HOME/bin:$PATH"
+
+COPY --from=jdk_installer /opt/jdk /opt/jdk
+
+COPY --from=graphite_installer /usr/local/lib/node_modules/@withgraphite /opt/graphite
+RUN ln -s /opt/graphite/graphite-cli/graphite.js /usr/local/bin/gt
+
+# Required to compress speech files in the tts.py script
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)